### PR TITLE
Fix compile with Sofa changes

### PR DESCRIPTION
**Pull request by EulalieCoevoet**
- MechanicalParams
- getEvaluator

WIP: drawAsEdges seems to have been removed from TetrahedronFEMForceField.
I don't know if you want to remove it too or add the option to the component HyperReducedTetrahedronFEMForceField.

### CODE REVIEW

File: src/component/forcefield/HyperReducedTetrahedronFEMForceField.inl [hunk 1]
```diff
@@ -462,7 +463,7 @@ void HyperReducedTetrahedronFEMForceField<DataTypes>::draw(const core::visual::V
 
     const VecCoord& x = this->mstate->read(core::ConstVecCoordId::position())->getValue();
 
-    const bool edges = (drawAsEdges.getValue() || vparams->displayFlags().getShowWireFrame());
+//    const bool edges = (drawAsEdges.getValue() || vparams->displayFlags().getShowWireFrame());
```
Comment (damienmarchal) [inline src/component/forcefield/HyperReducedTetrahedronFEMForceField.inl:466]: why ?

File: src/component/forcefield/HyperReducedTetrahedronFEMForceField.h [hunk 2]
```diff
@@ -161,7 +161,7 @@ class HyperReducedTetrahedronFEMForceField : public virtual TetrahedronFEMForceF
     using TetrahedronFEMForceField<DataTypes>::_gatherPt; //use in GPU version
     using TetrahedronFEMForceField<DataTypes>::_gatherBsize; //use in GPU version
     using TetrahedronFEMForceField<DataTypes>::drawHeterogeneousTetra;
-    using TetrahedronFEMForceField<DataTypes>::drawAsEdges;
+//    using TetrahedronFEMForceField<DataTypes>::drawAsEdges;
```
Comment (damienmarchal) [inline src/component/forcefield/HyperReducedTetrahedronFEMForceField.h:164]: This is related to: 
https://github.com/sofa-framework/sofa/commit/b2582f1600bf4d945d839cb6612da40f7d5a24a8

**Comment by olivier-goury** [comment]
> * MechanicalParams
> * getEvaluator
> 
> WIP: drawAsEdges seems to have been removed from TetrahedronFEMForceField.
> I don't know if you want to remove it too or add the option to the component HyperReducedTetrahedronFEMForceField.

Hello Eulalie! Thanks for the PR! Ideally, I want to do just the same that is done in TetrahedronFEMForcefield, so removing it seems fine.

File: src/component/forcefield/HyperReducedTetrahedronFEMForceField.h (hunk 2 above)
Comment (olivier-goury) [reply src/component/forcefield/HyperReducedTetrahedronFEMForceField.h:164]: Ok so the idea is that drawAsEdges was redundant with the wireFrame box in the GUI of SOFA?

**Comment by EulalieCoevoet** [comment]
Okay then, I removed it :)

**Review by olivier-goury — APPROVED** [review]
Thanks for the PR!

**Pull request merged**